webui/react/src/pages/TaskList.table.tsx

```diff
@@ -24,6 +24,7 @@ export const columns: ColumnType<CommandTask>[] = [
   },
   {
     key: 'name',
+    // render: added in TaskList.tsx
```

Comment:
took me a bit to remember we're overriding this somewhere else.

Reply:
sorry, and yes, that's a great idea to have a comment there